lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -282,6 +285,49 @@ class ReportUIFeatures {
     this._dom.resetTemplates();
   }
 
+  /* eslint-env browser */
+  /**
+   * Jumps to respective audit on page load if hash is provided in URL.
+   * Collapsed content is opened automatically.
+   * Element is highlighted via CSS selector.
+   */
+  _jumpToAudit() {
+    const hash = window.location.hash;
+    if (!hash) {
+      return;
+    }
+    const audit = this._document.querySelector(hash);
+    if (!audit || !audit.classList.contains('lh-audit')) {
+      return;
+    }
+    if (audit.firstElementChild && audit.firstElementChild.tagName === 'DETAILS') {
+      audit.firstElementChild.setAttribute('open', 'true');
+    }
+    this._openClosestDetails(audit);
+  }
+
+  /* eslint-env browser*/
```

Comment:
Let's keep this simpler than adding the whole env and remove this but add `HTMLDetailsElement` to the eslint `/* globals */` block at the top of the file